java/src/apps/GuiLafConfigPane.java

```diff
@@ -212,22 +184,29 @@ private int getDefaultFontSize() {
     static ActionListener listener;
 
     public void doFontSize(JPanel panel) {
-
+        GuiLafPreferencesManager manager = InstanceManager.getDefault(GuiLafPreferencesManager.class);
```

Comment:
Remove the manager variable, add the method
```
private GuiLafPreferencesManager manager() {
    return InstanceManager.getDefault(GuiLafPreferencesManager.class);
}
```
And then replace all uses of ```manager``` with ```manager()```